eeps/eep-0043.md

```diff
@@ -2,7 +2,7 @@
     Status:         Final/R17B  Implemented in OTP R17B (except ...)
     Type:           Standards Track
     Created:        04-Apr-2013
-    Erlang-Version: R17A
+    Erlang-Version: OTP_R17A
```

Comment:
We were using `OTP-RXXX` and then we switch to `OTP-19.0`
Is this change intended? Why not make everything `OTP-RXXX` and `OTP-XXX`.
Just curious to understand the change.

Reply:
```suggestion
    Erlang-Version: OTP-17.0
```

Releases before 17 were created using tags that look like this: `OTP_RXBY` and then with 17 we changed to `OTP-X.Y.Z` to be more compatible with how other opensource projects use versions.